packages/application-shell/package.json

```diff
@@ -18,12 +18,20 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./src/index.js",
-  "module": "./src/index.js",
+  "main": "dist/cjs/index.js",
+  "module": "dist/es/index.js",
```

Comment:
Shouldn't this be `dist/application-shell.cjs.js` and `dist/application-shell.es.js`

Reply:
Have updated this, now both cjs and es live in the same directory.